src/asymmetric_cipher.c

```diff
@@ -293,43 +300,41 @@ static int p11prov_rsaenc_decrypt(void *ctx,
 endsess:
     ret = f->C_CloseSession(session);
     if (ret != CKR_OK) {
-        P11PROV_raise(encctx->provctx, ret,
-                      "Failed to close session %lu", session);
+        P11PROV_raise(encctx->provctx,
+                      ret,
+                      "Failed to close session %lu",
+                      session);
     }
 
     return result;
-
 }
 
 static struct {
     CK_MECHANISM_TYPE type;
     unsigned int ossl_id;
     const char *string;
 } padding_map[] = {
-    { CKM_RSA_X_509, RSA_NO_PADDING, OSSL_PKEY_RSA_PAD_MODE_NONE },
-    { CKM_RSA_PKCS, RSA_PKCS1_PADDING, OSSL_PKEY_RSA_PAD_MODE_PKCSV15 },
-    { CKM_RSA_PKCS_OAEP, RSA_PKCS1_OAEP_PADDING, OSSL_PKEY_RSA_PAD_MODE_OAEP },
-    { CKM_RSA_X9_31, RSA_X931_PADDING, OSSL_PKEY_RSA_PAD_MODE_X931 },
-    { CK_UNAVAILABLE_INFORMATION, 0, NULL }
-};
+    {CKM_RSA_X_509, RSA_NO_PADDING, OSSL_PKEY_RSA_PAD_MODE_NONE},
+    {CKM_RSA_PKCS, RSA_PKCS1_PADDING, OSSL_PKEY_RSA_PAD_MODE_PKCSV15},
+    {CKM_RSA_PKCS_OAEP, RSA_PKCS1_OAEP_PADDING, OSSL_PKEY_RSA_PAD_MODE_OAEP},
+    {CKM_RSA_X9_31, RSA_X931_PADDING, OSSL_PKEY_RSA_PAD_MODE_X931},
+    {CK_UNAVAILABLE_INFORMATION, 0, NULL}};
 
```

Comment:
I think leaving spaces within braces make it more legible, but it is not a total deal breaker, is this configurable?